src/components/App.js

```diff
@@ -0,0 +1,10 @@
+import React, { Component } from 'react';
+
+class App extends Component {
+  render() {
+    return (
+      <div><h1>iReporter</h1></div>
```

Comment:
Good work @llwasampijja, however, this `h1` tag text is a little misleading, It would be better if we have the actual name of the project we are working with. `Author's Haven` preferably. Thank you.

Reply:
Welldone, @llwasampijja, also, format the div and h1 indentation to 2 spaces i.e.
```
<div>
  <h1>....</h1>
</div>
```